src/index.html

```diff
@@ -5,8 +5,264 @@
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product-image"
+        >
+        <h3 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__item-reviews">Reviews: 5</p>
+        <p class="card__item-price">Price:</p>
+        <p class="card__item-price-number">$2,199</p>
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product-image"
+        >
+        <h3 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__item-reviews">Reviews: 5</p>
+        <p class="card__item-price">Price:</p>
+        <p class="card__item-price-number">$2,199</p>
+        <a
+          href="#buy"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product-image"
+        >
+        <h3 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__item-reviews">Reviews: 5</p>
+        <p class="card__item-price">Price:</p>
+        <p class="card__item-price-number">$2,199</p>
+        <a
+          href="#buy"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product-image"
+        >
+        <h3 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="stars stars--4">
```

Comment:
Add empty lines between multiline sibling blocks of HTML
```suggestion
        <p class="card__product-code">Product code: 195434</p>
        
        <div class="stars stars--4">
```